ui/lib/sync/addon/components/secrets/page/overview.hbs

```diff
@@ -2,26 +2,26 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 }}
-{{#unless @isActivated}}
-  {{#if (or @licenseHasSecretsSync @isHvdManaged)}}
+{{#unless this.flags.secretsSyncIsActivated}}
```

Comment:
these services were already called in the component and I find args like this tedious to trace when they're passed down all the way from the route model.